docs/Configuring_the_native_Android_SIP_client.md

```diff
@@ -1,6 +1,8 @@
 These instructions detail how to enable the stock Android SIP client.
-These instruction are tested in a Samsung Galaxy Nexus running 4.2.2. If
-you get them working on another device, please add to the list below.
+These instruction are tested in a Samsung Galaxy Nexus running 4.2.2. For
+details on how to enable the SIP client on a Nexus 6 running 5.1.1 [see
+below](Configuring_the_native_Android_SIP_client.md#instructions-1).
```

Comment:
I'm worried it's not going to be maintainable to keep a separate list of nearly-identical instructions for every device - that might leave this document quite long, hard-to-follow and hard-to-maintain.

Instead I'd have some general instructions and then device-specific subpoints:

1. Bring up the internet calling accounts menu

    - On many devices, this is Call settings > Internet Call Settings > Accounts
    - On a Nexus 6 running 5.1.1, this is Calls > Calling Accounts > Internet Call Settings > Internet calling (SIP) accounts